guides/common/modules/snip_module-enable-note.adoc

```diff
@@ -0,0 +1,6 @@
+Enablement of the module `{dnf-module}` warns about a conflict with `postgresql:10` and `ruby:2.5` as these modules are set to the default module versions on {EL} 8.
```

Comment:
Should this file be named `dnf-module-enable`? Module is used so often that I think it should be qualified.

Reply:
yes